activejob/lib/active_job/test_helper.rb

```diff
@@ -308,9 +304,9 @@ def clear_performed_jobs # :nodoc:
 
         def enqueued_jobs_size(only: nil) # :nodoc:
           if only
-            enqueued_jobs.select { |job| Array(only).include?(job.fetch(:job)) }.size
+            enqueued_jobs.count { |job| Array(only).include?(job.fetch(:job)) }
```

Comment:
It's minimal enough. In the future though we generally ask for refactoring to be removed. It makes history difficult to reason about.